refactor: explanations for interfaces (#54)
aeternity/dex-contracts-v2: contracts/interfaces/README.md

> - The interfaces from ./for-export are autogenerated and are full contracts definition interface to be

Reviewer: ```suggestion
- The interfaces from ./for-export are auto-generated and are full contracts definition interface to be
```

used by sdk-clients outside this project in order to interact with the dex.

> In js project these can be accessed in the form of `var source = require('dex-contracts-v2/build/INTERFACE_NAME.aes')`

Reviewer: ```suggestion
In a JavaScript project these can be accessed in the form of `var source = require('dex-contracts-v2/build/INTERFACE_NAME.aes')`
```

> - The the purpose for the rest of the interfaces is to be used inside this project for tests (js) or inside the contracts.

Reviewer: ```suggestion
- The purpose of the rest of the interfaces is to use them inside this project for the tests (JS) or inside the contracts.
```

> We are defining here only the entrypoints that are used and sometime we splitt a contract entrypoints into different interface files 

Reviewer: ```suggestion
We are defining here only the entrypoints that are used and sometime we split a contract entrypoints into different interface files 
```

to avoid circular references (eg. IAedexV2Factory and IAedexV2FactoryForPair)

> More details you can find it at the top level of these interfaces

Reviewer: ```suggestion
You can find further details at the top of each of these interfaces
```
